services/app-web/src/components/SubmissionSummarySection/ContractDetailsSummarySection/ContractDetailsSummarySection.test.tsx

```diff
@@ -235,18 +235,20 @@ describe('ContractDetailsSummarySection', () => {
 
     it('does not render download all button when on previous submission', () => {
         renderWithProviders(
-            <Route
-                path={RoutesRecord.SUBMISSIONS_REVISION}
-                component={() => (
-                    <ContractDetailsSummarySection
-                        submission={draftContractAndRatesSubmission}
-                        submissionName="MN-PMAP-0001"
-                    />
-                )}
-            />,
+            <Routes>
+                <Route
+                    path={RoutesRecord.SUBMISSIONS_REVISION}
+                    element={
+                        <ContractDetailsSummarySection
+                            submission={draftContractAndRatesSubmission}
+                            submissionName="MN-PMAP-0001"
+                        />
+                    }
+                />
+            </Routes>,
             {
                 routerProvider: {
-                    route: '/submissions/15/revisions/2',
+                    route: '/submissions/15/summary/revisions/2',
```

Comment:
Why was this route changed?  To me this is a classic nested resource - submissions have many revisions. So `submission/:id/revisions/:id` make a lot of sense.

Reply:
I could also see `submission/:id/summary` redirecting to `submission/:id/revisions/:latest`.

Reply:
> Why was this route changed? To me this is a classic nested resource - submissions have many revisions. So submission/:id/revisions/:id makes sense.

Oops something I forgot to revert back from testing some alternative routing.😅